docs/docs/vulnerability/languages/java.md

```diff
@@ -0,0 +1,51 @@
+# Java
+
+## Features
+Trivy supports three types of Java scanning: `jar/war/par/ear`, `pom.xml` and `*gradle.lockfile` files.
+The following table provides an outline of the features Trivy offers.
+
+
+| Artifact         |  Maven repository [^1]   | Dev dependencies |
+|------------------|:------------------------:|:-----------------|
+| jar/war/par/ear  |            ✓             | Include          |
+| pom.xml          |            ✓             | Exclude          |
+| *gradle.lockfile |            -             | Exclude          |
+
+### Jar
+To find information about your jar[^2] file, Trivy parses `pom.properties` and `MANIFEST.MF` files in your jar[^2] file and takes `required properties`[^3].
+
+If those files don't exist or don't contain enough information - Trivy uses [maven repository REST API](https://central.sonatype.org/search/rest-api-guide) to try to get missing information.
+
```

Comment:
could you please make this small change (my OCD would be very happy 😄) "to try to get the missing information."

Reply:
Done